meson.build

```diff
@@ -1194,13 +1195,10 @@ if features['egl-x11']
 endif
 
 plain_gl = get_option('plain-gl').require(
-    get_option('libmpv') and gl_allowed,
-    error_message: 'libmpv was not enabled!',
+    gl_allowed,
```

Comment:
and IMO this should be kept, plain-gl has no useful effect for cplayer

Reply:
If cplayer and libmpv are built in one configuration, then cplayer will contain this code, too.